recipe/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "mclbn256" %}
+{% set version = "1.3.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nthparty/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 2087e21a431d49673f8d0bff5e1b6d073ec7e98c3c9ffa4e3fd18c8ec9068f74
+  patches:
+    - 0001-change-loadlibrary-path.patch
+
+build:
+  skip: True  # [s390x or py<37]
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  build:
+    - patch  # [not win]
+    - m2-patch  # [win]
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python
+    - mcl
+
+test:
+  imports:
+    - mclbn256
+  requires:
+    - pip
+  commands:
+    - pip check
+
```

Comment:
Could you please test the package (at least on `unix`)?
`- python -c "import mclbn256; mclbn256.mclbn256.assert_bilinearity(); mclbn256.mclbn256.assert_serializable(); print('pass')"`
See https://github.com/nthparty/mclbn256/blob/3a750c370526424dd988604b86b54e30b2da55de/.github/workflows/lint-test-build-upload.yml#L68C23-L68C146